### DISCUSSION TITLE
How to use image background in CustomTkinter

### DISCUSSION THREAD

**Discussion by sistemasolar5** [TomSchimansky/CustomTkinter#127] [Q&A]
Is there anyway of using images as backgrounds using CustomTkinter, I'm new on this, I already did this using Tkinter, but I don't know how to do it on CustomTkinter.

The screen shot is with Tkinter normally.

Thanks in advance, I tried this, of course is wrong, can you please help me on this.

photo = customtkinter.CTkCanvas(file = "/home/sistema/PycharmProjects/Pythonpruebas/venv/graficos/planets.png")
w = customtkinter.CTkCanvas(win, image="/home/sistema/PycharmProjects/Pythonpruebas/venv/graficos/planets.png")
w.pack()

![Screenshot from 2022-05-30 15-46-05](https://user-images.githubusercontent.com/106487116/171057941-4d70066c-7f48-4b05-b09e-bddb9f28a7f1.png)

**Comment by felipetesc** [comment]
Hello there! Default tkinter gui elements should work with CTK. Also, it's always good to indent the code, so we can just copy and test it right away. If you find a issue after trying to mix tkinter with ctk to add background in CustomTkinter, I will help you.

**Reply by sistemasolar5** [reply]
Thank you very much!!! I will try combine TK an CTK, and indend my code properly!!!

**Comment by felipetesc** [comment]
I just remembered, there is a sample here https://github.com/TomSchimansky/CustomTkinter/blob/master/examples/example_background_image.py

**Reply by felipetesc** [reply]
Hey? If example from the link above helped you, can close the question ? Thanks.
